DatadogRUM/Sources/Instrumentation/RUMInstrumentation.swift

```diff
@@ -80,23 +86,32 @@ internal final class RUMInstrumentation: RUMCommandPublisher {
         self.actionsHandler = actionsHandler
         self.uiApplicationSwizzler = uiApplicationSwizzler
         self.longTasks = longTasks
+        self.appHangs = AppHangsObserver(
+            appHangThreshold: appHangThreshold,
+            observedQueue: mainQueue,
+            dateProvider: dateProvider,
+            telemetry: telemetry
+        )
 
         // Enable configured instrumentations:
         self.viewControllerSwizzler?.swizzle()
         self.uiApplicationSwizzler?.swizzle()
         self.longTasks?.start()
+        self.appHangs?.start()
```

Comment:
enabled by default?

Reply:
This is TBD with the product, see [this ADR: Decision #2](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3473147398/ADR+-+App+Hangs+detection+in+iOS+SDK) (internal). This PR is against a feature branch, the final API will be added before it lands on `develop`.